tools/devops/automation/scripts/GitHub.psm1

```diff
@@ -256,6 +256,10 @@ class GitHubComments {
     }
 
     [bool] IsPR() {
+        if (-not $this.PRIds) {
+            return $false
+        }
+
         # if the object has a list of pr ids, we are a pr, else check the resource trigger
```

Comment:
This seems unrelated?

Reply:
Yes, I was testing something. And I pushed it here, was going to land in a diff pr and rebase